### ISSUE TITLE
.Net Wrapper - Issue using DashStyles

### ISSUE THREAD

**Issue by ksherman** [highcharts/highcharts#16101]
I've seen this reported a few times in Issues, this is still an ongoing bug for at least a year now near as I can tell. 

#### Expected behaviour
We are trying to set the dash styles according to the JS docs: https://api.highcharts.com/highcharts/yAxis.gridLineDashStyle

#### Actual behaviour
It looks like in v8.1.2.1, GridLineDashStyle was changed from a string to a Hashtable:
```
public Hashtable GridLineDashStyle { get; set; }
```
The challenge is, this appears to break what the JavaScript render is expecting: https://api.highcharts.com/highcharts/yAxis.gridLineDashStyle. We're getting render errors as the function `SVGElement.prototype.dashstyleSetter/1` is expecting a string.

#### Live demo with steps to reproduce
And see this diff on FuGet: https://www.fuget.org/packages/Highsoft.Highcharts/8.1.2.1/lib/netstandard2.0/diff/7.2.1.1/

This C# code:
```
// YAxis
chart.YAxis = new List<YAxis>
{
    new YAxis
    {
        Title = new YAxisTitle() { Text = "Dollar Amount" },
        MinorGridLineWidth = 0,
        MinorTickInterval = "auto",
        MinorTickLength = 2,
        MinorTickWidth = 1,
        TickLength = 10,
        TickWidth = 1,
        TickColor = "black",
        GridLineDashStyle = new Hashtable() { { "dashstyle", "Dash" } },
        GridLineColor = "#0606FF",
        Labels = new YAxisLabels() { Formatter = "function () {return '$' + Highcharts.numberFormat(this.value, 2)}" }
    }
};
```
Is rendering this (relevant) JS:
```
"yAxis": [
  {
    "minorGridLineWidth": 0.0,
    "tickWidth": 1.0,
    "labels": {
      "formatter": function () { return '$' + Highcharts.numberFormat(this.value,
        2)
      }
    },
    "gridLineColor": "#0606FF",
    "minorTickInterval": "auto",
    "tickColor": "black",
    "minorTickWidth": 1.0,
    "gridLineDashStyle": { //<---- Should just be "Dash"
      "dashstyle": "Dash" 
    },
    "title": {
      "text": "Dollar Amount"
    }
  }
]
```

#### Product version
v8.1.2.1 + (trying to use latest version)

**Comment by raf18seb** [comment]
Thanks for reporting @ksherman.

FYI @brofixo

**Comment by ksherman** [comment]
@brofixo Hey hey! Any update on releasing this fix you mentioned was in process on #14275 ?
